static/templates/settings/deactivate-realm-modal.handlebars

```diff
@@ -0,0 +1,13 @@
+<div id="deactivate-realm-modal" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivate_realm_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
```

Comment:
where is this close button?

Reply:
This is the "X" at the top-right corner of modal.
![close](https://user-images.githubusercontent.com/22238472/35582222-30dae0a6-0614-11e8-86c7-158e862c8b93.png) Cancel also does the same duty.

Reply:
ah, got it.